libs/watt/src/lib/components/button/watt-button.component.scss

```diff
@@ -54,47 +54,8 @@
 
 watt-button {
   display: inline-block;
-}
-
-.mat-button {
-  @include watt.typography-font-weight("semi-bold");
-  @include watt.typography-font-size("m");
-  @include watt.space-inset-squish-m;
-
-  &.mat-button {
-    min-width: 44px; // Magic UX width
-    height: 44px; // Magic UX normal height
-  }
-
-  watt-spinner {
-    position: absolute;
-  }
-
-  .mat-button-focus-overlay {
-    background: transparent; // Remove the overlay background color from Material Design
-  }
-
-  .mat-button-wrapper {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-
-    .content-wrapper {
-      display: flex;
-      gap: var(--watt-space-s);
-    }
-
-    .content-wrapper--loading {
-      visibility: hidden;
-    }
-
-    watt-icon {
-      margin: -5px; // to offset the icons inherent padding of 5px
-    }
-  }
```

Comment:
issue: the primary button changes to sharp corners on hover

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/21586ab1-adfd-43ec-a422-149fe32b2beb)